README.rst

```diff
@@ -9,20 +10,64 @@ Qt Style Sheet Inspector
 A inspector widget to view and modify style sheet of a Qt app in runtime.
 
 
-* Free software: MIT license
+Usage
+-----
 
+In order to use the inspector widget on your application, it's necessary to initialize the class :code:`style_sheet_inspector_class` passing the instance of the :code:`QMainWindow` from the application.
 
-Observation
------------
+The repository demo_qt_inspector_ contains a full example of a Qt Application with an inspector widget being called by a shortcut action.
+
+.. _demo_qt_inspector: https://github.com/williamjamir/demo_qt_inspector
+
+
+See the demo in action:
+
+.. image:: https://github.com/williamjamir/demo_qt_inspector/blob/master/images/qt_inspector_demo.gif
+    :width: 10px
+    :height: 10px
+    :scale: 10 %
 
-It need PyQt5 to work but it doesn't have it as a dependency, as testing with PyQt5 pypi proved
-unreliable (may be changed in the future).
 
 
 Features
 --------
+view current style sheet of application during runtime
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    The inspector only checks for style sheets that was applied to the QApplication, it's the topmost and any change here can be propagated to all child's. 
```

Comment:
Some typos here:

`that was applied` -> `that were applied`
`child'`s` -> `children`

Reply:
Commit https://github.com/ESSS/qt_style_sheet_inspector/pull/5/commits/11115fdbf8c963366ee8c58ccca8e79c425144a8 fix this issue ;)